src/Tribe/Editor.php

```diff
@@ -211,4 +211,17 @@ public function is_classic_editor() {
 
 		return $is_classic_editor_request || $disabled_by_plugin || $disabled_by_filter;
 	}
+
+	/**
+	 * Returns whether Blocks are being used for Events
```

Comment:
```suggestion
	 * Returns whether Blocks are being used for Events.
```

Full stop at end of comments.